mwibutsa/rich-form
# Project setup

#### Description of changes made

> This PR sets up initial project configurations including
   - React app initialization using create react app
   - Firebase
   - Readme file 
   - Pull request template
   - Eslint configuration

#### Type of change

- [x] Chore (changes which add some configurations)

#### How Has This Been Tested?


#### How should this be manually tested?

- After cloning the [repo](https://github.com/mwibutsa/rich-form.git)
- Check out this branch by `ch-project-setup`
- Open the app in your preferred editor and navigate through code files to check on configurations made

#### Checklist
